src/js/widgets/DateWidget.jsx

```diff
@@ -94,7 +95,7 @@ export default class DateWidget extends React.Component {
         {!monthYear && <div className="form-datefield-day">
           <label className="input-date-label" htmlFor={`${id}Day`}>Day</label>
           <select
-            autoComplete="false"
+            autoComplete={options.autocomplete || null}
```

Comment:
I'm pretty sure this will throw an axe error like the checkboxes. Please advise if you'd like confirmation and I can mock it on Codepen or other site for testing.

Reply:
Wow, that would be unfortunate since it would be handy to use `bday-month` or `bday-day` on these. Maybe that's one reason why USWDS 2.0 went back to text boxes for birthday month and day.